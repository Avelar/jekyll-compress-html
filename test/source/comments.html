---
layout: compress
---

<!--[if IE]><html class=\"ie\"><![endif]-->
<p>One</p>
<!-- Comment -->
<p>Two</p>
