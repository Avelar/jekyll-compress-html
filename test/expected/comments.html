<div><!-- One --><div>Two</div><!-- Three --><div>Four</div></div>
